clar2wasm/src/tools.rs

```diff
@@ -178,6 +181,108 @@ pub fn evaluate_at(
 pub fn evaluate(snippet: &str) -> Result<Option<Value>, ()> {
     evaluate_at(snippet, StacksEpochId::latest(), ClarityVersion::latest()).map_err(|_| ())
 }
+pub fn unicode_to_byte_sequence(input: Value) -> Value {
```

Comment:
I don't understand the purpose of this function. Why is it transforming the bytes in the `Value::Sequence(SequenceData::String(CharType::UTF8))` type?

Reply:
This function receives bytes in Unicode. Interpreter automatically converts those bytes, so I need to convert my expect result to bytes to match with interpreter/compiler result.

Reply:
This seems like the wrong approach. You should probably never have a utf8 `Value` in a format different from the one generated by `Value::string_utf8_from_bytes`.

Reply:
Alright. So since `Value::string_utf8_from_bytes` doesn't generate Unicode characters, how should I pass Unicode characters to interpreter as a part of a valid utf-8 string in clarity?

Reply:
> So since Value::string_utf8_from_bytes doesn't generate Unicode characters

I don't understand what you mean by this. There are several examples in this repo showing that function used with a range of unicode characters, e.g. 
```rs
Value::string_utf8_from_bytes("helŁo world 愛🦊".into())
```